R/logger.R

```diff
@@ -0,0 +1,3 @@
+
+# Setup logger namespace
+logger::log_layout(logger::layout_glue_colors, namespace = "shiny.telemetry")
```

Comment:
prettify logs with colors, optional change that's not really needed but improves quality of life for developers